packages/shared/providers.yaml

```diff
@@ -619,6 +620,22 @@ bamboohr-basic:
             pattern: '^[a-z0-9_-]+$'
             example: domain
             suffix: .bamboohr.com
+            order: 1
+    credentials:
+        username:
+            type: string
+            title: API key
+            description: The API Key of your BambooHR account
+            pattern: '^[a-zA-Z0-9]+$'
+            secret: true
+        password:
+            type: string
+            title: Password
+            description: Password
+            # https://documentation.bamboohr.com/docs/getting-started
+            # Bamboo HR is using basic auth with an api key
+            default_value: x
```

Comment:
nice

Reply:
yes, you'll be surprised Freshdesk is also doing this so the option was already there :D